src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -71,6 +71,29 @@ AttributRadiationFlag
 
 /*########################### define species #################################*/
 
+/*--------------------------- photons -------------------------------------------*/
+
+value_identifier(float_X, DummyMass, 1.0);
```

Comment:
what is the technical problem again with setting mass to zero or not defining it at all?